source/object-retention/minio-object-locking.rst

```diff
@@ -246,47 +246,35 @@ preferred SDK.
    .. tab-item:: MinIO Console
       :sync: console
 
-      Select the :guilabel:`Buckets` section of the MinIO Console to access
-      bucket creation and management functions. Select the bucket row from the
-      list of buckets. You can use the :octicon:`search` :guilabel:`Search` bar
-      to filter the list. 
+      Select the :guilabel:`Buckets` section of the MinIO Console to access bucket creation and management functions. You can use the :octicon:`search` :guilabel:`Search` bar to filter the list. 
       
       .. image:: /images/minio-console/console-bucket.png
          :width: 600px
          :alt: MinIO Console Bucket Management
          :align: center
 
-      From the :guilabel:`Bucket` view, look for the
-      :guilabel:`Retention` section and click :guilabel:`Enabled`. This section
-      is only visible if the bucket was created with object locking enabled.
+      Each bucket row has a :guilabel:`Manage` button that when selected opens the management view for that bucket.
 
-      .. image:: /images/minio-console/console-bucket-overview.png
+      .. image:: /images/minio-console/console-bucket-manage.png
          :width: 600px
          :alt: MinIO Console Bucket Management
          :align: center
 
-      From the :guilabel:`Set Retention Configuration` modal, set the 
-      desired bucket default retention settings.
+      From the :guilabel:`Retention` section and click :guilabel:`Enabled`.
```

Comment:
```
From the :guilabel:`Retention` section, click :guilabel:`Enabled`.
```

The `and` should be a comma instead.

Or perhaps:

```
Go to the :guilabel:`Retention` section and click :guilabel:`Enabeled`.
```

Reply:
Actually, the first option keeps things parallel with line 266.

Reply:
Done, and changed 'click' to 'select' to deal w/ non-mouse users